encord/orm/base_dto/base_dto_pydantic_v1.py

```diff
@@ -30,7 +31,8 @@ def from_dict(cls: Type[T], d: Dict[str, Any]) -> T:
             raise EncordException(message=str(e)) from e
 
     def to_dict(self, by_alias=True, exclude_none=True) -> Dict[str, Any]:
-        return self.dict(by_alias=by_alias, exclude_none=exclude_none)  # type: ignore[attr-defined]
+        # Pydantic v1 is missing the 'model_dump()' method, the below is suboptimal but works
+        return json.loads(self.json(by_alias=by_alias, exclude_none=exclude_none))  # type: ignore[attr-defined]
```

Comment:
Could you provide a reason for this change?

Reply:
`dict` is shallow in PydanticV1, to a point where it would output a dictionary with `UUID` values. And then the JSON serialiser would fail when trying to actually send the request.

This is why we're using `jsonable_encoder` all over the place in the BE; but it's part of FastApi and I don't feel like figuring out the license implications for copy-pasting that thing.

An alternative would be to do serialisation straight to the JSON string here and re-jigger the `post` to pass it straight to `requests` as `data`. The advantage of doing it the way I've done is that it also allows us to pass `UUID` or `datetime` GET parameters in a natural way.